scripts/rspec/rspec.ps1

```diff
@@ -173,29 +173,28 @@ function GenerateBaseClassIfSecondLanguage()
         $existingCSClassText = Get-Content -Path "${csRulesFolder}\\${csClassName}.cs" -Raw
         $existingVBClassText = Get-Content -Path "${vbRulesFolder}\\${vbClassName}.cs" -Raw
 
-        $oldCsClass ="    public sealed class ${csClassName} : SonarDiagnosticAnalyzer"
-        $newCsClass ="    public sealed class ${csClassName} : ${className}Base<SyntaxKind>"
-        $oldVbClass ="    public sealed class ${vbClassName} : SonarDiagnosticAnalyzer"
-        $newVbClass ="    public sealed class ${vbClassName} : ${className}Base<SyntaxKind>"
+        $oldCsClass ="public sealed class ${csClassName} : SonarDiagnosticAnalyzer"
+        $newCsClass ="public sealed class ${csClassName} : ${className}Base<SyntaxKind>"
+        $oldVbClass ="public sealed class ${vbClassName} : SonarDiagnosticAnalyzer"
+        $newVbClass ="public sealed class ${vbClassName} : ${className}Base<SyntaxKind>"
         $existingCSClassText = ReplaceTextInString -oldText $oldCsClass -newText $newCsClass -modifiableString $existingCSClassText
         $existingVBClassText = ReplaceTextInString -oldText $oldVbClass -newText $newVbClass -modifiableString $existingVBClassText
 
-        $diagnosticIdToken = "        private const string DiagnosticId = ""${ruleKey}"";"
+        $diagnosticIdToken = "    private const string DiagnosticId = ""${ruleKey}"";"
         $existingCSClassText = RemoveText -textToRemove $diagnosticIdToken -modifiableString $existingCSClassText
         $existingVBClassText = RemoveText -textToRemove $diagnosticIdToken -modifiableString $existingVBClassText
 
-        $messageFormatToken = "       private const string MessageFormat = """";"
+        $messageFormatToken = "   private const string MessageFormat = ""FIXME"";"
```

Comment:
Educational: Why did you add the FIXME there?

Reply:
So when we add a new rule, we can more clearly see that we have to change that